nexus/src/external_api/params.rs

```diff
@@ -315,13 +315,23 @@ pub struct NetworkInterfaceUpdate {
 
 // IP POOLS
 
+// Type used to identify a Project in request bodies, where one may not have
+// the path in the request URL.
+#[derive(Debug, Clone, Deserialize, Serialize, JsonSchema)]
+pub struct ProjectPath {
+    pub organization: Name,
+    pub project: Name,
+}
```

Comment:
This feels a lot like the global / non-global image split. Should we have different endpoints for the "global" vs "non-global" IP pools?

Reply:
Maybe? One difference I can see with images is that the IP Pools should still be limited to operators. I don't believe that really impacts the implementation of the authz checks, though. It would just be a different route, and a different way of getting the organization and project names. Even in that case, I believe you can still have a fleet-child object, it's just the route that's scoped. If that sounds right, I'm fine changing it. I don't have a strong preference, assuming only operators can muck with them.

Reply:
@rmustacc Any thoughts on this one?

Reply:
Also, to be clear, I think the project-scoped and top-level versions of this function would be identical. They'd literally call the same function, with an `Option<ProjectPath>` argument contained somewhere.

Reply:
I talked offline with Robert about this. The short answer is that it may make sense to have a "read-only view" of the route tree under `/ip-pools` also available under a project-scoped route. For example, this would make it easier for users (and probably in the implementation) to find the pools available in the context of a given project.

That said, I've filed https://github.com/oxidecomputer/omicron/issues/1470 to follow-up on it. I'm deferring because this probably depends a lot on how the top-level routes and operator API RFDs (288 and 290, respectively) shake out. We can revisit this then, when we have a better sense of how this will impact the few places this might come up in the API (looking at you, `/images`...).